project/src/main/java/com/google/sps/Models/User.java

```diff
@@ -24,4 +26,12 @@ public String getNickname() {
   public String getEmail() {
     return email;
   }
+
+  public ArrayList<String> getDocs() {
+      return docHashes;
+  }
+
+  public void setDocs(ArrayList<String> newHashes) {
```

Comment:
When do you set docs for a certain user? I would think addDoc will be a more useful API.

Reply:
Right! AddDoc is definitely more useful, setDocs was something else we were trying, its not used so I will delete it.